include/ddc/kernels/splines/bsplines_non_uniform.hpp

```diff
@@ -75,23 +85,20 @@ class NonUniformBSplines : detail::NonUniformBSplinesBase
         int m_nknots;
 
     public:
+        /// @brief The type of the discrete dimension representing the B-splines.
         using discrete_dimension_type = NonUniformBSplines;
 
+        /// @brief The type of a discrete domain identifying the B-splines.
         using discrete_domain_type = DiscreteDomain<DDim>;
 
+        /// @brief The type of a discrete element identifying a B-spline.
         using discrete_element_type = DiscreteElement<DDim>;
 
+        /// @brief The type of a discrete vector representing an "index displacement" between two B-splines.
         using discrete_vector_type = DiscreteVector<DDim>;
 
         Impl() = default;
 
-        template <class OriginMemorySpace>
-        explicit Impl(Impl<DDim, OriginMemorySpace> const& impl)
-            : m_domain(impl.m_domain)
-            , m_nknots(impl.m_nknots)
-        {
-        }
-
         /// @brief Construct a `Impl` using a brace-list, i.e. `Impl bsplines({0., 1.})`
         explicit Impl(std::initializer_list<ddc::Coordinate<Tag>> breaks)
             : Impl(breaks.begin(), breaks.end())
```

Comment:
You need to specify what this brace-list represents

Reply:
bae41cc7d22ed5464a816d64484655ad14d2bd10